web/src/ui/commons.js

```diff
@@ -0,0 +1,50 @@
+const Select = ({
```

Comment:
Machen wir bei Komponenten doch besser `function Select({ ... }`, das schaut mir etwas komisch aus mit der Formatierung :D

Reply:
Das ist blöd mit dem speichern dann, Aktuell ist es so, dass er bei jedem mal speichern alles formatiert, weißt du ob ich das irgendwie in den prettier einstellungen machen kann? Ansonsten mach ich den "format on Save" raus und formatiere dann bevor ich pushe